pkg/proxy/namespace/namespace.go

```diff
@@ -0,0 +1,14 @@
+package namespace
+
+import "net/url"
+
+// Namespace represents a namespace in a member cluster for the specific sso user-workspace
+type Namespace struct {
+	Username    string
+	ClusterName string
+	APIURL      url.URL
+	Namespace   string
+	Workspace   string
```

Comment:
same here: this field is never used

Reply:
https://github.com/codeready-toolchain/registration-service/pull/224#discussion_r745990333